cmd/registry/core/tasks.go

```diff
@@ -37,7 +37,7 @@ func Worker(ctx context.Context, taskChan <-chan Task) {
 	for task := range taskChan {
 		err := task.Run()
 		if err != nil {
-			log.Printf("%s: %s", task.Name(), err.Error())
+			log.Printf("%s: %s", task.String(), err.Error())
```

Comment:
FYI this is equivalent to `log.Printf("%s: %s", task, err)`. 

From the [fmt docs](https://golang.org/pkg/fmt/#hdr-Printing): 
> If the format (which is implicitly %v for Println etc.) is valid for a string (%s %q %v %x %X), the following two rules apply:
>
>4. If an operand implements the error interface, the Error method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).
>
>5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any).

Reply:
Also, not relevant to this PR in particular but still worth mentioning: the log should have a newline character since Printf doesn't include one by default. Providing more info could also be nice since it's not ending execution, e.g. `log.Printf("Error running task %q: %s\n", task, err)`.

Reply:
Thanks! I made the simplification change that you suggested above, but want to address the error messaging recommendation separately because I found some other error messages being produced by tasks and think we should handle them consistently. Also in my usage of log.Printf, I've found that it automatically adds a newline at the end of log messages, which seems related to [this](https://golang.org/src/log/log.go?s=10139:10169#L373).